app/lib/search_service.dart

```diff
@@ -28,6 +28,14 @@ SearchService get searchService => ss.lookup(#_search);
 /// Register a new [SearchService] in the current service scope.
 void registerSearchService(SearchService s) => ss.register(#_search, s);
 
+abstract class CseTokens {
+  static const pageMapDocument = 'document';
+  static const detectedTypePrefix = 'dt_';
+  static const experimentalScore = 'exp_score';
+
+  static String detectedType(String type) => 'dt_$type';
```

Comment:
Consider using detectedTypePrefix, since you added it, instead of repeating `dt_` ?

(Also maybe add `String` type to the constants)